client/src/components/Card/Card.js

```diff
@@ -0,0 +1,19 @@
+import styled from 'styled-components/macro'
+// import styled from "styled-components";
```

Comment:
Please remove unused and commented out code if you don't need it!

Reply:
Will do